tests/test_main.py

```
@@ -1572,6 +1572,60 @@ def test_run_main_with_log_level_override(
 
+    @patch("mmrelay.main.print_banner")
+    @patch("mmrelay.config.load_config")
+    @patch("mmrelay.config.load_credentials")
+    @patch("mmrelay.config.is_legacy_layout_enabled")
+    @patch("mmrelay.config.get_base_dir")
+    @patch("mmrelay.config.get_data_dir")
+    @patch("mmrelay.config.get_log_dir")
+    @patch("mmrelay.config.os.makedirs")
+    def test_run_main_legacy_layout_warning(
+        self,
+        mock_makedirs,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_makedirs`

(ARG002)